components/nexus-common/src/main/java/org/sonatype/nexus/common/node/LocalNodeAccess.java

```diff
@@ -12,6 +12,12 @@
  */
 package org.sonatype.nexus.common.node;
 
+import java.security.cert.Certificate;
+
+import org.sonatype.sisu.goodies.lifecycle.Lifecycle;
+
+// TODO: rename to LocalNodeIdentity ?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)